docs/reference/modules/remote-clusters.asciidoc

```diff
@@ -1,29 +1,37 @@
-[[modules-remote-clusters]]
+[[remote-clusters]]
 == Remote clusters
-
 You can connect a local cluster to other {es} clusters, known as _remote
-clusters_. Once connected, you can search remote clusters using
-<<modules-cross-cluster-search,{ccs}>>. You can also sync data between clusters
-using <<xpack-ccr,{ccr}>>.
-
-To register a remote cluster, connect the local cluster to nodes in the
-remote cluster using one of two connection modes:
-
-* <<sniff-mode,Sniff mode>>
-* <<proxy-mode,Proxy mode>>
-
-Your local cluster uses the <<modules-network,transport layer>> to establish
-communication with remote clusters. The coordinating nodes in the local cluster
-establish <<long-lived-connections,long-lived>> TCP connections with specific
-nodes in the remote cluster. {es} requires these connections to remain open,
-even if the connections are idle for an extended period.
-
-You can use the <<cluster-remote-info, remote cluster info API>> to get
-information about registered remote clusters.
+clusters_. Remote clusters can be located in different datacenters or 
+geographic regions, and contain indices or data streams that can be replicated
+or searched by a local cluster. You can use remote clusters in {ccr} and {ccs}.
```

Comment:
I like where you're headed with this change. I'll leave out the links because they exist in the following paragraphs, but I'll update to read:
>Remote clusters can be located in different datacenters or  geographic regions, and contain indices or data streams that can be replicated with {ccr} or searched by a local cluster using {ccs}.